viam-cartographer.go

```diff
@@ -254,7 +254,25 @@ func (cartoSvc *cartographerService) GetInternalStateStream(ctx context.Context,
 	ctx, span := trace.StartSpan(ctx, "viamcartographer::cartographerService::GetInternalStateStream")
 	defer span.End()
 
-	return grpchelper.GetInternalStateStreamCallback(ctx, name, cartoSvc.clientAlgo)
+	return grpchelper.GetInternalStateCallback(ctx, name, cartoSvc.clientAlgo)
+}
+
+// GetPointCloudMap creates a request, calls the slam algorithms GetPointCloudMap endpoint and returns a callback
+// function which will return the next chunk of the current pointcloud map.
+func (cartoSvc *cartographerService) GetPointCloudMap(ctx context.Context, name string) (func() ([]byte, error), error) {
```

Comment:
[note to self] have to add these now, before adding to slam interface, so that viam-cartographer still satisfies slam interface when we bump rdk version here (or could have added these when bumping rdk version here)